QuizApp/Scenes/HomePage/Extensions/HomeVCExtension.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  HomeVCExtension.swift
```

Comment:
Please correct the file name according to the ViewController it contains

Reply:
HomeViewControllerExtension is long name so let's leave it like this